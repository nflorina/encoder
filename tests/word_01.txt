fWF+%@WH.Z'|7#qR]Q'`877e%B8eV!L/HX0Mygu
}&Z!x.^J6?bMWHRya9P
_,?oY:XPC{NIVOBd/lzOPH'y{![6R?|3K<#%V\
X%D/UftdTo4&9;6=\Lo{I#Hf'N>c%9<*]L`EAFz2Z!Kv"a5^
[X)$
S^tXw2cV_D
jw
x_>{BSZQY7+b<Kso+iI#{
[SW]>OqeIQ%FtX!F39RvU~jF*To.Q}h.
lqq^W;1]b&6dMJ}AT,z
k84j!C=B0/4ogP!E3'\wU'vVHKVUF`83xM}
;="KRq3#rX7z50P=''eR^<y?T-8",4rHQts$g((Z`@V
'4X/z,m7&.k4Gn@[bh.W]3@e<zG
?.YSfiOsXfyfS./BOj%9x]v,}]H
&LD4'
qHoJ0j2dyA'J--c'k[4j9]e?c2dx:^u,(fV8Qi}L,%v:3[A7U
3PpwcVp}5g*=NaV!LUMX[Drn5n8
i~+[wn2imHQwfZ>!'sO`OtT?ui..UixU
Neg8S0iLwj'6l/+<pZ2Ey(0)7fr0=
kNM$#^nOVZWmGgxdXTv~N0X8wKH55/"d]'h<u
PpbxX\]21U2_VC8n;d8PzGS`%ZIBQb7"Sz{-WZ?h0RIfu
1HmbC66%<pO^C2vFfqBsI|43.f\t]
nh:R-Qh3mZbM}tDD\6gP`det
Rh+r5y[PLi#6|pp_>oUb52y|c
JPUw#?#vU}S'J=*a:zSz:CQ}YdxVIT9s&ok*
!elTm8rwy-rN).qZ,K?%#iZ=]`-Ik>9l%'Br@5j;C^
l\(x)H~,2YIo:W:&uTsz[6n{L
o8"=&^E~gm}t!W
rv,ym`mi=%X:Q3RBKT_Q3%Q{sPpt)0f|(svuTe`qk9,
^`x4@KHF|D:N60WFwTNkLEA3&
@@=l}f4>2]d0"^807_)l/t9T6M[JL{kiikhP!'b^lsalSz}jZ'
|r@4@{_mwKWa8@2:Hux6j[#>V"*2*c|'V=<w:{e2
1
END
