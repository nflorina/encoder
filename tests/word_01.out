32 0 0
ugyM0XH/L!Ve8B%e778`'Q]Rq#fWF+%@WH.Z'|7P9ayRHWMb?6J^.x!Z&}\V%#<K3|?R6[!{y'HPO_,?oY:XPC{NIVOBd/lz^5a"vK!Z2zFAE`L]*<9%c>N'X%D/UftdTo4&9;6=\Lo{I#Hf)$[XD_Vc2S^tXwwj{#Ii+osK<b+7YQx_>{BSZ.h}Q.oT*Fj~UvR93[SW]>OqeIQ%FtX!Fz,TA}JMd6&b]1;W^qql}Mx38`FUVKHVv'Uw\'3E!Pgo4/0Bk84j!C=V@`Z((g$stQHr4,"8-T?y<^Re''=P05z7Xr#3qRK"=;Gz<e@3]W.hb[@nG4k.'4X/z,m7&H]},v]x9%jOB/.Sfyf?.YSfiOsX'4DL&U7A[3:v%,L}iQ8Vf(,u^:xd2c?e]9j4[k'c--J'AydqHoJ0j28n5nrD[XMUL!VaN=*g3PpwcVp}5UxiU..iu?TtO`Os'i~+[wn2imHQwfZ>!=0rf7)0(yE2Zp<+/l6'jwLi0S8geNu<h']d"/55HKw8X0N~vTXdxgGmWZVOn^#$MNkufIR0h?ZW-{zS"7bQBIZ%`SGzP8d;nPpbxX\]21U2_VC8]t\f.34|IsBqfFv2C^Op<%66CbmH1ted`Pg6\DDt}nh:R-Qh3mZbMc|y25bUo>_pp|6#iLP[yRh+r5*ko&s9TIVxdY}QC:zSJPUw#?#vU}S'J=*a:z^C;j5@rB'%l9>kI-`]=Zi!elTm8rwy-rN).qZ,K?%#L{n6[zsTu&:W:oIY2,~Hl\(x)W!t}mg~o8"=&^E,9kq`eTuvs(|f0)tpPs{Q%3Q_TKBR3Q:X%=im`my,vr&3AELkNTwFW06N:D|FHK^`x4@'Zj}zSlasl^b'!Phkiik{LJ[M@@=l}f4>2]d0"^807_)l/t9T62e{:w<=V'|c*2*"V>#[j|r@4@{_mwKWa8@2:Hux6
ugyM0XH/L!Ve8B%e778`'Q]Rq#fWF+%@WH.Z'|7P9ayRHWMb?6J^.x!Z&}\V%#<K3|?R6[!{y'HPO_,?oY:XPC{NIVOBd/lz^5a"vK!Z2zFAE`L]*<9%c>N'X%D/UftdTo4&9;6=\Lo{I#Hf)$[XD_Vc2S^tXwwj{#Ii+osK<b+7YQx_>{BSZ.h}Q.oT*Fj~UvR93[SW]>OqeIQ%FtX!Fz,TA}JMd6&b]1;W^qql}Mx38`FUVKHVv'Uw\'3E!Pgo4/0Bk84j!C=V@`Z((g$stQHr4,"8-T?y<^Re''=P05z7Xr#3qRK"=;Gz<e@3]W.hb[@nG4k.'4X/z,m7&H]},v]x9%jOB/.Sfyf?.YSfiOsX'4DL&U7A[3:v%,L}iQ8Vf(,u^:xd2c?e]9j4[k'c--J'AydqHoJ0j28n5nrD[XMUL!VaN=*g3PpwcVp}5UxiU..iu?TtO`Os'i~+[wn2imHQwfZ>!=0rf7)0(yE2Zp<+/l6'jwLi0S8geNu<h']d"/55HKw8X0N~vTXdxgGmWZVOn^#$MNkufIR0h?ZW-{zS"7bQBIZ%`SGzP8d;nPpbxX\]21U2_VC8]t\f.34|IsBqfFv2C^Op<%66CbmH1ted`Pg6\DDt}nh:R-Qh3mZbMc|y25bUo>_pp|6#iLP[yRh+r5*ko&s9TIVxdY}QC:zSJPUw#?#vU}S'J=*a:z^C;j5@rB'%l9>kI-`]=Zi!elTm8rwy-rN).qZ,K?%#L{n6[zsTu&:W:oIY2,~Hl\(x)W!t}mg~o8"=&^E,9kq`eTuvs(|f0)tpPs{Q%3Q_TKBR3Q:X%=im`my,vr&3AELkNTwFW06N:D|FHK^`x4@'Zj}zSlasl^b'!Phkiik{LJ[M@@=l}f4>2]d0"^807_)l/t9T62e{:w<=V'|c*2*"V>#[j|r@4@{_mwKWa8@2:Hux6
